### DISCUSSION TITLE
Dual cursor on screen

### DISCUSSION THREAD

**Discussion by meffewmeffew** [ppy/osu#22476] [Q&A]
### Type

Cosmetic

### Bug description

I have had this issue with lazer for several versions of lazer. Its impact is enough that it often makes gameplay unplayable.
When the osu cursor nears the top of the screen, the windows (Windows 10 Pro) cursor appears on top of osu client.

Things I've noticed about the bug:
-clicking with my mouse while the dual cursors are on screen results in a minimizing of the osu client, as if it were alt-tabbed
-the bug only happens in fullscreen mode
-the dual cursor at top of the screen corresponds to my windows taskbar being at the top of the screen (I believe)
-the bug will happen in menus as well as during gameplay.
-the footage below shows the osu using the horizontal and vertical scale feature. However the bug still happens when using 100% vertical and 100% horizontal.
-no setting changes done to my windows taskbar has stopped this issue (that I know or have looked up)
-upon starting osu lazer, the bug is not there, however after a few seconds the bug starts to show up.

I hope this bug gets looked into and fixed. I really really love lazer, but this one issue is making it nearly unplayable for me. Best of luck, and if there are any other questions I'd be happy to answer them.

### Screenshots or videos

https://user-images.githubusercontent.com/123708819/214994270-51913174-a222-4764-95dd-8e70e7ef2e8e.mp4

### Version

2023.123.0

### Logs

no logs, this is a cosmetic issue

**Comment by peppy** [comment]
This is likely something else you're running on your PC causing the issue. Could you try shutting down all applications (even things in system tray) and see if you can still replicate?
